src/Tribe/Aggregator/Settings.php

```diff
@@ -44,7 +44,7 @@ public function __construct() {
 	 * @param WP_Screen $screen
 	 */
 	public function maybe_clear_eb_credentials( $screen ) {
-		if ( 'tribe_events_page_tribe-common' !== $screen->base ) {
```

Comment:
Just added a couple methods in common and others on the TEC Admin\Settings class so that we can get this without hardcoding it: https://github.com/the-events-calendar/the-events-calendar/pull/3880/files#diff-e4d0aa550c265b7f74fd03910dfff06ef7ae724f65875f8d05f9e4d48badf217R159